examples/hello/index.php

```diff
@@ -15,10 +15,30 @@
  * limitations under the License.
  */
 
+/**
+ * To use this, set the following environment variables:
```

Comment:
Nit: can these also be specified as command-line args?

Reply:
there is no CLI for php at the moment